src/components/screens/InfoCard.js

```diff
@@ -67,6 +67,24 @@ const contactView = (contact) => (
   </View>
 );
 
+const mergeWebUrls = ({ webUrls, contact, contacts }) => {
+  const mergedWebUrls = [...webUrls];
```

Comment:
is `webUrls` ensured to not be `undefined` here?
trying to spread `undefined` causes an error.

you can avoid it with
```
const mergedWebUrls = [...(webUrls ?? [])];
```
or with
```
const mergedWebUrls = webUrls ? [...webUrls] : [];
```
of which i prefer the second version for readability reasons.

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/170/commits/d919904fe6442110d647dfd00b57a9d280e445f2